### PR TITLE
Babel preset for Jotai

### PR DESCRIPTION
**Pull request by Thisen**
Adds a `babel` preset for `jotai`, making it easier for people to use our existing plugin `plugin-debug-label` and future plugins.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "build:zustand": "rollup -c --config-zustand",
     "build:redux": "rollup -c --config-redux",
     "build:urql": "rollup -c --config-urql",
-    "build:babel": "rollup -c --config-babel_plugin-debug-label",
+    "build:babel": "rollup -c --config-babel_preset && rollup -c --config-babel_plugin-debug-label",
```
Comment (dai-shi) [inline package.json:51]:
```suggestion
    "build:babel:preset": "rollup -c --config-babel_preset",
    "build:babel:plugin-debug-label": "rollup -c --config-babel_plugin-debug-label",
```
Hope it works.

Comment (Thisen) [reply package.json]: Good idea 👍🏼

**Comment by dai-shi** [comment]
It's working!
image

**Review by dai-shi — APPROVED** [review]
LGTM
I'll leave this open for a while and merge it later.

**Pull request merged**